circle.yml

```diff
@@ -33,13 +33,31 @@ jobs:
       - run:
           name: Clean gradle
           command: ./gradlew clean
+      - run:
+          name: Add preferences to gradle.properties file
+          command: |
+            rm -fr gradle.properties
```

Comment:
why are removing this file, afaik it won't exist when doing a checkout?

Reply:
Left over from other experimentation. Removed.